### ISSUE TITLE
Planning Milestone v0.2

### ISSUE THREAD

**Issue by draychev** [openservicemesh/osm-health#118]
This GitHub Issue is to be used for planning OSM Health release v0.2

Please comment with recommendations on GitHub Issues or general areas that we should bundle with the OSM Health v0.2 release.

**Comment by draychev** [comment]
- [ ] #117 
- [x] #116 
- [x] #115 
- [ ] #109 
- [ ] #107

**Comment by johnsonshi** [comment]
Yeah I've got 2 PRs (that I need to rebase) that fixes 2 of these @draychev
